source/default/_patterns/02-molecules/tabs/_tabs.twig

```diff
@@ -0,0 +1,53 @@
+{#
+/**
+ * @file
+ * tabs
+ *
+ * Available variables for tabs:
+ *  - tabs.label [string]: Used for aria-label for tabslist.
+ *  - tabs.show_title_in_panel [boolean]: Shows title in panel.
+ *  - tabs.data [array]: array of tab objects.
+ *    - title [string]: Used for Tab title and Header inside of tab panel.
+ *    - panel [string]: Panel content.
+ */
+#}
+{# Use seed to ensure that each group of tabs have unique IDs, needed if multiple tabs are on a single page. #}
+{% set seed = random() %}
+<div class="tabs mb-4">
+  <ul class="tabs__list px-1" role="tablist" {{ tabs.label ? "aria-label='#{tabs.label}'" }}>
+    {% for item in tabs.data %}
+      {% set item_uid = "#{seed}-#{loop.index0}" %}
+      {% set tab_id = "tabs-tab--#{item_uid}" %}
+      {% set panel_id = "tabs-panel--#{item_uid}" %}
+      <li class="tabs__list-item inline-block" role="presentation">
+        <a class="tabs__list-link inline-block py-1 px-8 text-black bg-gray-300 border border-gray-500 border-b-0 rounded-t hover:bg-white hover:border-blue-600 hover:underline hover:text-blue-600 focus:border-blue-600 focus:outline-none focus:underline focus:text-blue-600"
+          id="{{ tab_id }}"
+          href="#{{ panel_id }}"
+          role="tab"
+          aria-controls="{{ panel_id }}"
+          aria-selected="false">
+          {{ item.title }}
+        </a>
+      </li>
+    {% endfor %}
+  </ul>
+
+  <div class="tabs__panels">
+    {% for item in tabs.data %}
+      {% set item_uid = "#{seed}-#{loop.index0}" %}
+      {# A11y show initially let js choose to which to show/hide. #}
+      <section class="tabs__panels-item border border-gray-500 p-4 bg-white focus:border-blue-600 focus:outline-none"
```

Comment:
Is it worth placing these attributes into an array and allowing `attributify` filter handle these?